### PR TITLE
Cleanup reader-gtfs dependencies

### PR DESCRIPTION
**Pull request by otbutz**
Moved versions into dependencyManagement section of the parent POM

### CODE REVIEW

**Comment by otbutz** [comment]
Fixes #2207

**Comment by otbutz** [comment]
@karussell any objections?

File: reader-gtfs/pom.xml [hunk 1]
```diff
@@ -96,7 +88,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <!-- Currently we need a bit more memory than for other tests -->
                 <configuration>
-                    <argLine>-Xmx1200m -Xms1200m</argLine>
+                    <argLine>-Xmx1200m -Xms1200m -Duser.language=en</argLine>
```
Comment (karussell) [inline reader-gtfs/pom.xml:91]: Why is this necessary?

**Comment by karussell** [comment]
@michaz can you have a quick look into this?

File: reader-gtfs/pom.xml (hunk 1 above)
Comment (otbutz) [reply reader-gtfs/pom.xml:91]: There are unittests checking error message contents which break with a different locale. We already set this in the parent POM but we need to explicitly set it here again, as we're overriding the inherited configuration.

File: pom.xml [hunk 2]
```diff
@@ -136,6 +136,38 @@
...
+	        <dependency>
+	            <groupId>org.mapdb</groupId>
+	            <artifactId>mapdb</artifactId>
+	            <version>1.0.8</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>io.leonard</groupId>
+	            <artifactId>gtfs-realtime-bindings</artifactId>
+	            <version>0.0.7</version>
+	            <exclusions>
```
Comment (easbar) [inline pom.xml:158]: The 'official' repository now seems to be: https://github.com/MobilityData/gtfs-realtime-bindings

```
This project was originally created by Google - MobilityData started to maintain the project in early 2019.
```

I don't know what io.leonard is, but they recommend these coordinates here: https://github.com/MobilityData/gtfs-realtime-bindings/blob/master/java/README.md#add-the-dependency

```xml
<dependency>
  <groupId>io.mobilitydata.transit</groupId>
  <artifactId>gtfs-realtime-bindings</artifactId>
  <version>0.0.7</version>
</dependency>
```

**Review by easbar — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -136,6 +136,38 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+	        <dependency>
+	            <groupId>net.sourceforge.javacsv</groupId>
```
Comment (easbar) [inline pom.xml:140]: Looks like the indentation is slightly off here

File: pom.xml (hunk 2 above)
Comment (otbutz) [reply pom.xml:158]: https://search.maven.org/search?q=a:gtfs-realtime-bindings

`io.mobilitydata.transit` does not provide a 0.0.7 release

Comment (easbar) [reply pom.xml:158]: But it seems to be here: https://mvnrepository.com/artifact/io.mobilitydata.transit/gtfs-realtime-bindings/0.0.7 ?

Comment (easbar) [reply pom.xml:158]: Hm, but it does not work. See also this comment: https://github.com/MobilityData/gtfs-realtime-bindings/pull/70#issuecomment-726234803

Comment (otbutz) [reply pom.xml]: I've switched to 0.0.5. We can still update the lib in a later PR

Comment (otbutz) [reply pom.xml]: > But it seems to be here: https://mvnrepository.com/artifact/io.mobilitydata.transit/gtfs-realtime-bindings/0.0.7 ?

> Note: this artifact is located at JCenter repository (https://jcenter.bintray.com/)

Comment (easbar) [reply pom.xml]: Ok sure, 0.0.5 seems to use protobuf 3.6.1

**Comment by easbar** [comment]
Just a minor thing: Can you rather merge master into your feature branch than rebasing it onto current master? That makes it a bit easier to keep track of the changes.

**Comment by otbutz** [comment]
Force of habit. My employer enforces a rebase workflow :sweat_smile:

File: reader-gtfs/pom.xml (hunk 1 above)
Comment (easbar) [reply reader-gtfs/pom.xml:90]: ok

**Comment by easbar** [comment]
It can be useful to see the clean rebased feature branch, but rebasing a public branch also can be problematic and most of all the GitHub UI seems to not handle it very well. It now shows 'change requested', but the link to the requested change is broken... Anyway :)

**Comment by easbar** [comment]
Thanks!
